.github/release-drafter.yml

```diff
@@ -1,7 +1,7 @@
 # Inherit base settings from https://github.com/jenkinsci/.github/blob/master/.github/release-drafter.yml
 _extends: .github
 name-template: MinIO Plugin v$RESOLVED_VERSION
-tag-template: minio-$RESOLVED_VERSION
+tag-template: $RESOLVED_VERSION
```

Comment:
I seem to have made the wrong assumption fixing this yesterday, but based on your tag history you simply have only the version number as the tag. Now this should work

Reply:
Additionally, it might be a good idea to crate a empty GH release for your latest tag (e.g. tag: 1.1.0, releaseName: MinIO Plugin v1.1.0, description: ...) just so Release Drafter wont freak out and can create the next release properly :wink: 

(u could check what it found and drafted for the next release by checking the release view in GH as u can see the drafts there too)

Reply:
Ah okay, thanks for the suggestion. I'll take a look at that 🙂

Reply:
BTW, not sure how much u read the JenkinsCI readme on release drafter, if you want PR changes categorized in the changelog u will need to label them with the [supported labels](https://github.com/jenkinsci/.github/blob/master/.github/release-drafter.yml)